codebuild/release/validate-staging.yml

```diff
@@ -0,0 +1,32 @@
+version: 0.2
+# 'validate-staging' is an AWS::CodeBuild::Project that validates a CodeArtifact staging before a sonatype release
+
+env:
+  variables:
+    REGION: us-east-1
+    DOMAIN: crypto-tools-internal
+    REPOSITORY: java-ddbec-staging
+  parameter-store:
+    ACCOUNT: /CodeBuild/AccountId
+
+phases:
+    runtime-versions:
+      java: $JAVA_ENV_VERSION
+  pre_build:
+    commands:
+      - export SETTINGS_FILE=$(pwd)/codebuild/release/settings.xml
+      - export CODEARTIFACT_TOKEN=$(aws codeartifact get-authorization-token --domain $DOMAIN --domain-owner $ACCOUNT --query authorizationToken --output text --region ${REGION})
+      - export CODEARTIFACT_REPO_URL=https://${DOMAIN}-${ACCOUNT}.d.codeartifact.${REGION}.amazonaws.com/maven/${REPOSITORY}
```

Comment:
We should `cd examples` here to ensure we are only running `mvn verify` on this module, since it is that module which is actually testing pulling down the jar and using it as a dependency. Otherwise this will also build/test the `sdk1` module, which is unnecessary for this codebuild.

Reply:
Should that `cd` be in the **pre_build** or **build**? I assume **build**.

Reply:
Either one. For consistency with the ESDK release, I'm slightly biased towards pre_build https://github.com/aws/aws-encryption-sdk-java/blob/master/codebuild/release/validate-staging.yml#L20-L21

Reply:
Huh... re-reading this... I am not dead sure that the examples directory will even be present... We have not pulled down the repo or anything...